sdk/core/test/integration/otp/index.test.ts

```diff
@@ -311,6 +313,28 @@ describe('CommonNetworkMember (flows that require OTP)', () => {
     expect(responseError.name).to.eql('COR-17')
   })
 
+  it('#passwordlessLogin with custom messages ', async () => {
+    const stamp = `${Date.now()} stamp`
+    const delimiter = ':'
+    const messageParameters: MessageParameters = {
+      message: `Your verification code is ${delimiter}${stamp}${delimiter}{{CODE}}.`,
+      subject: `${stamp}${delimiter}{{CODE}}`,
+    }
+
+    const fullOptions = getOptionsForEnvironment(true)
+    const tag = `${fullOptions.env}_passwordlessLogin_integration`
+    const userName = TestEmailHelper.generateEmailForTag(tag)
+    const token = await CommonNetworkMember.passwordlessLogin(userName, fullOptions, messageParameters)
+    await wait(DELAY)
+    const [{ text, subject }] = await TestEmailHelper.getEmailsForTag(tag)
+    // eslint-disable-next-line no-unused-vars
+    const [_, expectedStampFromMessage, __] = text.split(delimiter)
+    const [expectedStampFromSubject, otp] = subject.split(delimiter)
+    expect(expectedStampFromMessage).to.equal(stamp)
+    expect(expectedStampFromSubject).to.equal(stamp)
```

Comment:
Maybe it would be more clear to extract the OTP code from subject, and then verify that
```
expect(text).to.equal(messageParameters.text.replace('{{CODE}}', otpCode));
expect(subject).to.equal(messageParameters.subject.replace('{{CODE}}, otpCode));
```
?

But it's up to you; that's just a suggestion.

Reply:
On this stage we do not have {{code}} we have real otp

Reply:
I mean we can:
1) Extract OTP code;
2) Substitute it into our templates;
3) Check that the actual message and subject match the expected ones.

Instead of extracting stamp and checking that stamp matches (ignoring everything else).